src/main/java/controllers/CreationBridge.java

```diff
@@ -0,0 +1,10 @@
+package controllers;
+
+import java.util.ArrayList;
+
+/**
+ * Defines a controller that handles the creation of a game.
+ */
+public interface CreationBridge {
```

Comment:
What would implement the bridges?

Reply:
Not included in this PR, but a controller.GameController (not the ui.windows one).